plugins/parsers/json/parser.go

```diff
@@ -114,6 +137,7 @@ func (p *JSONParser) switchFieldToTag(tags map[string]string, fields map[string]
 
 func (p *JSONParser) Parse(buf []byte) ([]telegraf.Metric, error) {
 
+	//if json_query is specified
```

Comment:
Remove this comment, it doesn't provide any new information on why and the what can be determined by looking at the code.